src/gui/synthesispopover.cpp

```diff
@@ -493,19 +512,86 @@ void SynthesisPopover::initUI() {
     _stackedWidget->insertWidget(DriveInfoClient::SynthesisStackedWidgetActivity, _notImplementedLabel2);
 
     // Bottom
-    BottomWidget *bottomWidget = new BottomWidget(this);
+    auto *bottomWidget = new BottomWidget(this);
     mainVBox->addWidget(bottomWidget);
 
+    //// Locked app
+    auto *lockedAppVesrionVBox = new QVBoxLayout(_lockedAppVesrionWidget);
```

Comment:
typo :)

Reply:
Well see :), here is the correction: [3c4d53e](https://github.com/Infomaniak/desktop-kDrive/pull/193/commits/3c4d53eee9b6ba9d5135c9a7a55430d1578416de)